File: Calculator.Api/Controllers/CalculatorController.cs
using Calculator.Api.Models;
using Calculator.Api.Services;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;
//using System.Web.Mvc;

namespace Calculator.Api.Controllers
{
    public class CalculatorController : ApiController
    {
        private readonly ICalculatorService _calculatorService;
        public CalculatorController(ICalculatorService calculatorService)
        {
            _calculatorService = calculatorService;
        }

        [HttpGet]
        public IHttpActionResult Calculate([FromUri]  CalculateParams param)
        {
            var result = _calculatorService.Calculate(param.Sign, param.FirstValue, param.SecondValue);
            var resultData = new ResultData
            {
                Result = result,
                ErrorMessage = string.Empty
            };
            return Ok(resultData);
        }
    }
}

File: Calculator.Api/UI/controllers/calculatorcontroller.js
(function (app) {
    var calculatorController = function ($scope, calculatorservice) {
        var vm = this;
        vm.result = 0;
        vm.sign = '+';
        vm.values = {
            first: 0,
            second: 0
        };
       
        vm.doCalculations = function (sign, a, b) {
            var data = {
                sign: sign,
                firstValue: a,
                secondValue: b
            };
            calculatorservice.calculate(data).then(x => {
                vm.result = x.data.Result;
            });
        };
    };
    app.controller("calculatorController", ["$scope", "calculatorService", calculatorController]);
}(angular.module("calculatorApp")));